CoffeeChat/Controller/GroupsViewController.swift

```diff
@@ -52,27 +90,57 @@ class GroupsViewController: UIViewController {
         backButton.addTarget(self, action: #selector(backButtonPressed), for: .touchUpInside)
         view.addSubview(backButton)
 
+        displayedGroups = groups
```

Comment:
Can we just do this above when you make displayedGroups?

Reply:
I get an error saying `Cannot use instance member 'groups' within property initializer; property initializers run before 'self' is available`

Reply:
Ah okay I see